Changelog update for upcoming 10.0.9

PVince81 opened the pull request:
Update changelog

Commit:
Changelog update for upcoming 10.0.9

phil-davis commented on CHANGELOG.md:
```diff
@@ -5,17 +5,56 @@ All notable changes to this project will be documented in this file.
 ### Removed
 ### Fixed
+- Improve user-sync command help description - [#31691](https://github.com/owncloud/core/issues/31691)
+- Fix deletion of files in trashbin in some partial selection scenarios - [#31700](https://github.com/owncloud/core/issues/31700)
```
Also fixes restore of partial selection

phil-davis commented on CHANGELOG.md:
```diff
@@ -5,17 +5,56 @@ All notable changes to this project will be documented in this file.
+- Do not set the password again if it hasn't changed - [#31370](https://github.com/owncloud/core/issues/31370)
+- Use correct l10n to translate 'password was changed' email - [#31553](https://github.com/owncloud/core/issues/31553)
+- Improve text in settings/personal App Password - [#31539](https://github.com/owncloud/core/issues/31539)
+- Fix default langage code example - [#31448](https://github.com/owncloud/core/issues/31448)
```
s/langage/language

phil-davis reviewed (commented):
Just little comments - text is generally fine.

PVince81 commented:
I've updated the changelog based on review comments and also added more of the stuff that got merged.
For the object store backport I rather mentionned the original master PRs as they had more information than the big backport PR.

@DeepDiver1975

phil-davis commented on CHANGELOG.md:
```diff
@@ -27,13 +32,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Bump symfony 3.4.9 to 3.4.11 - [#31571](https://github.com/owncloud/core/issues/31571)
 - Update jsdoc requirement to ~3.5.5 - [#30036](https://github.com/owncloud/core/issues/30036)
 - Removed example theme which now lives in the [theme-example repository](https://github.com/owncloud/theme-example) - [#31447](https://github.com/owncloud/core/issues/31447)
+- A user member of multiple groups is now excluded from sharing if at least one of their group is configured for exclusion - [#31737](https://github.com/owncloud/core/issues/31737)
```
A user who is a member of...

PVince81 replied:
fixed

PVince81 commented:
squashed
